Tutorials_UsingAPIs/Alarm/main.cpp

```diff
@@ -48,12 +45,10 @@ void set_time_leds(void)
 void inc_delay(void)
 {
     if (select_state == 0) {
-        delay += HOUR;
-        hour_count++;
+        delay += hour_inc++;
         hour_led = !hour_led;
     } else {
-        delay += MINUTE;
-        min_count++;
+        delay = +min_inc++;
```

Comment:
Typo here. Not `+=` like above. But then I think it's still wrong

Surely this and the above should look like.

```
delay += 1min;
min_inc++;
```

It's still a bit wonky, because there's no compelling reason to track both `delay` and `min_inc`. And why even bother making `min_inc` be Chrono if it's just a counter?

So, instead, you could forget the `delay` increment, just do `min_inc++` and have

```
auto delay = hour_inc + min_inc;
```

in `main` - making the Chrono nature of those two variables useful.

Reply:
@kjbracey-arm thanks for the feedback! I've cleaned it up now and tested the changes to make sure everything works as expected.